packages/create-app/tsconfig.json

```diff
@@ -1,16 +1,16 @@
 {
   "extends": "../../configurations/tsconfig.json",
-  "include": ["./src/**/*.ts"],
+  "include": ["./src/**/*.ts", "plopfile.ts"],
```

Comment:
Would it make sense to have this file inside the template and delete it after the app has been created?